webapp/security/views.py

```diff
@@ -268,8 +268,53 @@ def api_create_notice():
 # ===
 def cve_index():
 
-    list_cve = db_session.query(CVE).limit(10)
-    return flask.render_template("security/cve/index.html", list_cve=list_cve)
+    page = flask.request.args.get("page", default=1, type=int)
```

Comment:
Would you mind using `limit` and `offset` as per https://pages.apigee.com/rs/apigee/images/api-design-ebook-2012-03.pdf ?

Reply:
Delete this, as discussed